lib/workers/repository/errors-warnings.ts

```diff
@@ -49,14 +46,30 @@ export function getDepWarnings(
               if (!warnings.includes(message)) {
                 warnings.push(message);
               }
-              if (!warningFiles.includes(file.packageFile!)) {
-                warningFiles.push(file.packageFile!);
+              if (
+                file.packageFile &&
+                !warningFiles.includes(file.packageFile)
+              ) {
+                warningFiles.push(file.packageFile);
               }
             }
           }
         }
       }
     }
+  } catch (err) {
+    // istanbul ignore next
```

Comment:
please add a test instead

Reply:
I think we should remove the `try catch` because the catch block seems unreachable.

Reply:
sounds good